build-in-container/run.sh

```diff
@@ -0,0 +1,76 @@
+#! /bin/bash
+
+help() {
+    echo "------------ Mariner Build-in-Container ------------"
+    echo "
+    The run.sh script presents these options
+    -t                  creates container image
+    -b [repo_dir]       creates container,
+                        builds the specs under $repo_dir/SPECS/
+                        and places the output under $repo_dir/out/
+    -i [repo_dir]       create an interactive Mariner build container
+    -c [repo_dir]       cleans up Mariner workspace at $repo_dir, container images and instances
+    --help              shows help on usage
+
+    * unless provided, repo_dir defaults to the directory containing build-in-container tool
+    "
+    echo "----------------------------------------------------"
+}
+
+create_container() {
+    echo "Creating Container Image"
+    source ${tool_dir}/create-build-container.sh
+}
+
+build_mariner() {
+    echo "Creating Mariner Build Container and building Mariner SPECS"
+    if [ "$#" -ne 0 ]
+    then
+        repo_dir="$1"
```

Comment:
This needs to be converted into an absolute path (if you pass ~/... or ./... etc. docker will get angry).
```suggestion
        repo_dir="$(realpath $1)"
```